README.md

```diff
@@ -125,6 +125,54 @@ If you're converting existing ViewHolders, you can reference any named views (e.
 
 You can also mix and match `BindableItem` and other `Items` in the adapter, so you can leave legacy viewholders as they are by making an `Item<MyExistingViewHolder>`.  
 
+
+### Item with LiveData :
+
+The `ViewHolder` is a `LifecycleOwner` so you can use `LiveData` with Databinding
+
+```kotlin
+pubic class LiveDataItem extends BindableItem<ItemLiveDataBinding> {
+
+    // Simple wrapper around liveData
+    public class DataWrapper {
+        private MutableLiveData<String> data = new MutableLiveData<>();
+
+        public MutableLiveData<String> getData() {
+            return data;
+        }
+    }
+
+    public DataWrapper internalDataWrapper = new DataWrapper();
+
+    public LiveDataItem() {
+        super();
+    }
+
+    @Override public int getLayout() {
+        return R.layout.item_live_data;
+    }
+
+    @Override public void bind(@NonNull ItemLiveDataBinding viewBinding, int position) {
+        viewBinding.setWrapper(internalDataWrapper);
+    }
+}
+```
+
+or simply by observe the data of the LiveData in the `bind` method and update the view 
+
+```kotlin
+class LiveDataItem(private val data: LiveData<String>) : Item() {
+
+    override fun bind(viewHolder: ViewHolder, position: Int) {
+        data.observe(viewHolder, Observer {
```

Comment:
I could be wrong, but this example (not the system) seems like this could lead to some bad scenarios where:
1) A given Item's LiveData is being observed by multiple ViewHolders and could update all of them
2) A given ViewHolder is bound to multiple Item's LiveData sources (including more than once to the same source)

There is a way these issues can be resolved by ensuring that each ViewHolder's previous observers are removed on each rebind*, although it is a manual process that I'm not sure can be easily enforced within the library, similar to RxJava2's Disposables needing to be manually handled.

To illustrate the issue:
Imagine a scenario where we're displaying a simple list of LiveDataItems, and the RecyclerView could only show one Item at a time and had a buffer size of 1 (meaning that the exact same ViewHolder is recycled everytime).

1) Item 1 is displayed.
ViewHolder1 is created, and everything works fine.
2) The user scrolls to Item2.
VH1 is recycled and bound to Item2. Now exists two observers on both Item1 and Item2, each updating VH1. VH1 displays Item2's data. 
3) Item1 updates, but the user doesn't scroll
All of a sudden, VH1, which is supposed to be displaying Item2's data, is updated to display Item1's new data. Oh no!
4) User scrolls back up to Item1
VH1 is rebound to Item1, and now three observers exist! Two observers on Item1, and one on Item2, all updating VH1. VH1 displays Item1's data.
_branch 5a and 5b_
5a) Item2 updates
VH1 displays Item2's data while still indexed on Item1
5b) Item1 updates
VH1 is updated twice to display Item1's new data (not so bad, but still undesirable)

Things only continue to get worse the more the user scrolls.

Reply:
If you take  look inside the Item class you will see that I unbind the observer by changing the status of the lifecycleOwner (which will work with databinding too)

Reply:
Hmm, you may be on to something by using Lifecycle state changes to automatically unbind, although according to the [LiveData documentation](https://developer.android.com/topic/libraries/architecture/livedata), you probably need to set the unbound VH state to DESTROYED instead of back to CREATED.

> You can register an observer paired with an object that implements the LifecycleOwner interface. This relationship allows the observer to be removed when the state of the corresponding Lifecycle object changes to DESTROYED.